src/PerfView/SupportFiles/UsersGuide.htm

```diff
@@ -3827,7 +3827,18 @@ <h2>
         <li>
             The <strong><a id="CommandToRunTextBox">Command TextBox</a></strong> - This is only
             active for the Run command.&nbsp; It is the command line to run after collection
-            has been turned on.&nbsp;&nbsp; This textbox is inactive for the Collect command.&nbsp;
+            has been turned on.&nbsp;&nbsp; This textbox is hidden for the Collect command.&nbsp;
+        </li>
+        <li>
+            The <strong><a id="FocusProcessTextBox">Focus Process TextBox</a></strong> - This is only
```

Comment:
Please state before anything else that the focus process is either a decimal process ID or a name of an executable without the Directory part  but WITH the suffix.  (e.g. MyProgram.exe).

Also please put validation in that checks that it is one of these things  and produce an error immediately (rather than simply failing to collect anything which is what would otherwise happen).   

Other than that, I think it looks good.  

Thanks for doing this.

Reply:
Requested changes are done and pushed here..